src/tox/config/cli/parser.py

```diff
@@ -366,6 +368,12 @@ def add_color_flags(parser: ArgumentParser) -> None:
         choices=["yes", "no"],
         help="should output be enriched with colors, default is yes unless TERM=dumb or NO_COLOR is defined.",
     )
+    parser.add_argument(
+        "--stderr-color",
+        default="RED",
+        choices=["0", *Fore.__dict__.keys()],
+        help="color for stderr output, use 0 to disable coloring.",
```

Comment:
We already have NO_COLOR env-var users should use if they don't want color, let's not add 0.

Reply:
Here is the deal: I want color, I just don't want any specific color! Clearly using NO_COLOR is not suitable here.

Any other ideas? -- I an easily look for other string values like "NONE" instead of "0".

Reply:
Sorted.